app/src/main/java/hr/foi/air/servicesync/ui/components/ReservationCalendar.kt

```diff
@@ -0,0 +1,37 @@
+package hr.foi.air.servicesync.ui.components
+
+import androidx.compose.material3.DatePicker
+import androidx.compose.material3.DatePickerDialog
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.SelectableDates
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.rememberDatePickerState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import hr.foi.air.servicesync.R
+import hr.foi.air.servicesync.business.PresentAndFutureSelectableDates
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ReservationCalendar(selectedDates: SelectableDates, onDismiss: () -> Unit, onConfirm : (Long?) -> Unit) {
```

Comment:
Made calendar in a way it's compatible for future use. Although it wasn't in the issue, I must thank you for this.

Reply:
Glad to do it.